src/tools/TimeTools.h

```diff
@@ -27,6 +27,16 @@ namespace timeTools
 	*/
 	std::string convertMicrosecondUnixTimestampToIso(const long long & ts);
 
+	/**
+	* Convert an Unix timestamp which is given in seconds into a date in ISO format.
+	*/
+	__declspec(dllexport) std::string convertUnixTimestampToIso(const long long & ts);
```

Comment:
Do we really need to export that to final users?
I hope no.
If we really need, we should reuse export/import macro.

Reply:
Fixed.